fixtures/american-dj/hex100.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "HEX100",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["K"],
+    "createDate": "2023-04-22",
+    "lastModifyDate": "2023-04-22"
+  },
+  "links": {
+    "manual": [
+      "https://d295jznhem2tn9.cloudfront.net/ItemRelatedFiles/11302/ADJ%20MOD%20HEX100%20User%20Manual%20.pdf"
+    ],
+    "productPage": [
+      "https://www.adj.com/mod-hex100"
+    ]
+  },
+  "physical": {
+    "DMXconnector": "3-pin and 5-pin"
+  },
```

Comment:
QLC+ requires physical dimensions to create a fixture (based on what I see in the export test from GH Actions)